app/src/server/tasks/evaluateTestSetEntry.task.ts

```diff
@@ -99,14 +95,14 @@ export const evaluateTestSetEntry = defineTask<EvaluateTestSetEntryJob>({
       model: fineTune
         ? `openpipe:${fineTune.slug}`
         : COMPARISON_MODEL_NAMES[modelId as ComparisonModel],
-      messages: prunedMessages,
```

Comment:
For fine tunes we do the pruning within `getCompletion2` anyway so this is redundant. And for comparison models, we don't actually want the strings to be pruned, so this is unnecessary.